platform_main_feature.views_default.inc

```diff
@@ -1816,6 +1816,51 @@ function platform_main_feature_views_default_views() {
+  $handler->display->display_options['fields']['mods_identifier_non-marc-call-number_ms']['element_label_colon'] = FALSE;
+  /* Field: Islandora Solr: mods_originInfo_dateIssued_s */
+  $handler->display->display_options['fields']['mods_originInfo_dateIssued_s']['id'] = 'mods_originInfo_dateIssued_s';
+  $handler->display->display_options['fields']['mods_originInfo_dateIssued_s']['table'] = 'islandora_solr';
+  $handler->display->display_options['fields']['mods_originInfo_dateIssued_s']['field'] = 'mods_originInfo_dateIssued_s';
+  $handler->display->display_options['fields']['mods_originInfo_dateIssued_s']['label'] = 'Date';
+  $handler->display->display_options['fields']['mods_originInfo_dateIssued_s']['element_label_colon'] = FALSE;
+  $handler->display->display_options['fields']['mods_originInfo_dateIssued_s']['hide_empty'] = TRUE;
+  $handler->display->display_options['fields']['mods_originInfo_dateIssued_s']['link_to_object'] = 0;
+  /* Field: Islandora Solr: bpl_mods_physicalDescription_extent_concatenated_unit_ms */
+  $handler->display->display_options['fields']['bpl_mods_physicalDescription_extent_concatenated_unit_ms']['id'] = 'bpl_mods_physicalDescription_extent_concatenated_unit_ms';
+  $handler->display->display_options['fields']['bpl_mods_physicalDescription_extent_concatenated_unit_ms']['table'] = 'islandora_solr';
+  $handler->display->display_options['fields']['bpl_mods_physicalDescription_extent_concatenated_unit_ms']['field'] = 'bpl_mods_physicalDescription_extent_concatenated_unit_ms';
+  $handler->display->display_options['fields']['bpl_mods_physicalDescription_extent_concatenated_unit_ms']['label'] = 'Contents';
+  $handler->display->display_options['fields']['bpl_mods_physicalDescription_extent_concatenated_unit_ms']['element_label_colon'] = FALSE;
+  $handler->display->display_options['fields']['bpl_mods_physicalDescription_extent_concatenated_unit_ms']['hide_empty'] = TRUE;
+  /* Field: Islandora Solr: bpl_mods_abstract_all_types_mt */
+  $handler->display->display_options['fields']['bpl_mods_abstract_all_types_mt']['id'] = 'bpl_mods_abstract_all_types_mt';
+  $handler->display->display_options['fields']['bpl_mods_abstract_all_types_mt']['table'] = 'islandora_solr';
+  $handler->display->display_options['fields']['bpl_mods_abstract_all_types_mt']['field'] = 'bpl_mods_abstract_all_types_mt';
+  $handler->display->display_options['fields']['bpl_mods_abstract_all_types_mt']['label'] = 'Description';
+  $handler->display->display_options['fields']['bpl_mods_abstract_all_types_mt']['element_label_colon'] = FALSE;
+  $handler->display->display_options['fields']['bpl_mods_abstract_all_types_mt']['hide_empty'] = TRUE;
+  $handler->display->display_options['fields']['bpl_mods_abstract_all_types_mt']['link_to_object'] = 0;
+  /* Field: Islandora Solr: PID */
+  $handler->display->display_options['fields']['PID']['id'] = 'PID';
```

Comment:
Paranoia perhaps, but... this isn't going to collide with the contextual filter, is it?: https://github.com/discoverygarden/platform_main_feature/pull/280/files#diff-150c68da1c367e4dec73520c10bdbd10R1806

... would think it would also be available to be sub'd in...

... also, looks like there's only the one display for this... any reason not to "apply to all displays", instead of overriding for this one?... just... now the other set of unused config there, now: https://github.com/discoverygarden/platform_main_feature/pull/280/files#diff-150c68da1c367e4dec73520c10bdbd10R1769-R1804

Reply:
Looks like it should be fine... it uses different mechanisms for replacement of the contextual filters (`%1` or `!1`, kind of thing?), rather similar to https://github.com/discoverygarden/platform_main_feature/pull/280/files#diff-150c68da1c367e4dec73520c10bdbd10R1818

... should the same thing just be done for this footer link?

Reply:
Fixing the features export. Applied to all instead of just "This Block". It does look much cleaner now.

I'm unsure what the appropriate action is for the other point or if there is an action we want to take?

Reply:
... it's only the source of the value that's different... as in, the contextual filter corresponds to whatever's being passed to the view to render, whereas the `[PID]` is from the _result_ of the query for the given contextual filter... the result should only be the one same value, it's just getting it in a roundabout fashion... shrug? Doesn't _really_ matter I don't think, is just rather indirect.